recipe/meta.yaml

```diff
@@ -32,8 +32,7 @@ test:
 
 about:
   home: http://www.perl.org/
-  license: Perl Artistic
-  license_family: Other
+  license: Artistic-1.0-Perl
```

Comment:
Can we list the `license_file` here as well?

Reply:
Sure, I didn't include it as I assumed it was defaulting to LICENSE

Reply:
Ah, I think I understand now, this keyword specifies that the license is injected into the conda package, it has nothing to do with the LICENSE.txt in this feedstock, is this correct?

Reply:
that's right

Reply:
We might need to do a different thing for Windows as this repackages [Strawberry Perl]( http://strawberryperl.com ) there. Could use a selector to separate how the UNIX and Windows cases are handled.

Reply:
This should do the trick ( https://github.com/conda-forge/perl-feedstock/pull/42/files#r478641858 )